docs/source/loading.mdx

```diff
@@ -54,11 +54,19 @@ If you don't specify which data files to use, [`load_dataset`] will return all t
 
 </Tip>
 
-You can also load a specific subset of the files with the `data_files` parameter. The example below only loads files that match the grep pattern:
+You can also load a specific subset of the files with the `data_files` or `data_dir` parameter. Datasets are loaded from the Hub if you pass a relative path to `data_dir`. The relative path is transformed to a URL corresponding to where the dataset is loaded from. If you want to load a dataset from a local directory, you need to pass an absolute path instead:
```

Comment:
I think this may need some rephrasing: I would say it makes not much sense to pass an absolute local directory to load a Hub dataset *without a loading script*: I think this specific use case is only applicable to Hub datasets *with a loading script*.

I mean, passing `load_dataset("allenai/c4",...` just analyzes all the files in the repo to see that they are mainly JSON files and then choose the "json" builder.

Therefore, 
```python
load_dataset("allenai/c4", data_dir="/absolute/path/to/local/allenai/c4/en")
```
is equivalent to:
```python
load_dataset("json", data_dir="/absolute/path/to/local/allenai/c4/en")
```